css-pseudo-4/Overview.bs

```diff
@@ -1634,6 +1634,32 @@ Styling form control pickers: the ''::picker()'' pseudo-element</h3>
 &lt;/style>
   </pre>
 
+<h3 id="select-arrow">
+Select dropdown icon: the ''::select-arrow'' pseudo-element</h3>
+
+  The ''::select-arrow'' pseudo-element only matches when the <a>originating
+  element</a> is a <{select}> element.
+
+  ''::select-arrow'' is a [=part-like pseudo-element=], so there is no
+  restriction on which properties apply to the ''::select-arrow'' pseudo-element.
+
+  ''::select-arrow'' generates a box as if it was an immediate child of its
+  <a>originating element</a>, preceding any boxes generated by the ''::before''
+  pseudo-element, with content as specified by 'content'.
+
+<h3 id="check">
+Option checkmark icon: the ''::check'' pseudo-element</h3>
+
+  The '::check'' pseudo-element only matches when the <a>originating
+  element</a> is a <{option}> element.
```

Comment:
As I commented in the chromium prototype CL, should this instead say that the pseudo-element is only generated for option elements? That's an observable difference for `getComputedStyle(div, '::check')`.

Reply:
I changed it to say generated instead of matches